test/runtests.jl

```diff
@@ -245,7 +246,8 @@ end
 @test sprint(JSON.print, [Inf]) == "[null]"
 
 # Check printing of more exotic objects
-@test sprint(JSON.print, sprint) == string("\"function at ", sprint.fptr, "\"")
+# Test broken in v0.5, code is using internal structure of Function type!
+#@test sprint(JSON.print, sprint) == string("\"function at ", sprint.fptr, "\"")
```

Comment:
How can I track down the version string for that?  I know the commit number: 9bfd27bd380124174a5f37c342e5c048874d71a4, but I don't know how to go from there to:
`if VERSION < v"0.5.0-dev+1343"`.  As far as adjusting it in a general way, Jeff has said it shouldn't even be trying to print out the function pointer for JSON.